packages/carbon-graphs/README.md

```diff
@@ -1,20 +1,112 @@
 # Carbon Graphs
 
 [![Build Status](https://travis-ci.com/cerner/terra-graphs.svg?branch=main)](https://travis-ci.com/cerner/terra-graphs)
+![npm](https://img.shields.io/npm/v/@cerner/carbon-graphs)
 
-A graphing library built using d3 based on Cerner design standards.
+A vanilla JavaScript graphing library built using d3 based on Cerner design standards.
 
+<br>
+
+
+- [Installing Carbon](#installing-carbon)
 - [Getting Started](#getting-started)
-- [Documentation](../terra-graphs/docs/README.md)
+- [Graph Types](#graph-types)
 - [LICENSE](#license)
 
+## Installing Carbon
+
+- Install from [npmjs](https://www.npmjs.com/package/@cerner/carbon-graphs): 
+```sh
+npm i @cerner/carbon-graphs --save-dev
+```
+
 ## Getting Started
 
-- Install from [npmjs](https://www.npmjs.com): 
+To create a graph with carbon, first let's create a graph configuration object and use it to initialize that canvas:
+
+```javascript
+const GRAPH_DATA = {
+  bindTo: "#root",
+  axis: {
+    x: {
+      show: true,
+      label: "x-axis label",
+      lowerLimit: 0,
+      upperLimit: 100
+    },
+    y: {
+      show: true,
+      label: "y-axis label",
+      lowerLimit: 0,
+      upperLimit: 10,
+    },
+  }
+};
+
+var canvas = Carbon.api.graph(GRAPH_DATA);
 ```
-npm i @cerner/carbon-graphs --save-dev
+
+Let's create a sample dataset to plot:
+```javascript
+const CONTENT_DATA1 = {
+  key: "uid_1",
+  label: {
+    display: "Dataset A"
+  },
+  values: [
+    {
+      x: 10,
+      y: 9
+    },
+    {
+      x: 45,
+      y: 3
+    },
+    {
+      x: 60,
+      y: 7
+    },
+    {
+      x: 77,
+      y: 8
+    },
+    {
+      x: 94,
+      y: 2
+    }
+  ]
+};
 ```
 
+The data can now by plotted by as a line graph by using the following:
+
+```javascript
+    canvas.loadContent(Carbon.api.line(CONTENT_DATA1));
+```
+
+This results in the following graph:
+
+![Alt](../terra-graphs/docs/assets/carbon-simple-line-graph.png "simple line graph")
```

Comment:
The image is not displaying

![image](https://user-images.githubusercontent.com/62761486/101827275-b865a600-3af5-11eb-9e71-f15b26c8bd8d.png)

Reply:
If you click on `view file`, the image should be working there:

<img width="1285" alt="CleanShot 2020-12-10 at 22 11 33@2x" src="https://user-images.githubusercontent.com/38024451/101861012-ba028e80-3b34-11eb-8be6-4e600c55f041.png">

<img width="1129" alt="CleanShot 2020-12-10 at 22 13 26@2x" src="https://user-images.githubusercontent.com/38024451/101861121-f3d39500-3b34-11eb-9166-0e23ea4432c9.png">